docs/DeveloperGuide.md

```diff
@@ -540,8 +534,6 @@ testers are expected to do more *exploratory* testing.
        Draconian`, `link --friend`, ...
```

Comment:
There is inconsistency with these `...`, some are differently formatted. Perhaps they are not even necessary. Can you help to remove or make consistent?

Reply:
I agree, I shall remove them all.